bower.json

```diff
@@ -6,7 +6,7 @@
   "issues": "https://github.com/adaptlearning/adapt_framework/issues/new?title=contrib-bookmarking%3A%20please%20enter%20a%20brief%20summary%20of%20the%20issue%20here&body=please%20provide%20a%20full%20description%20of%20the%20problem,%20including%20steps%20on%20how%20to%20replicate,%20what%20browser(s)/device(s)%20the%20problem%20occurs%20on%20and,%20where%20helpful,%20screenshots.",
```

Comment:
while you're here, would you mind trimming the querystring from the issues URL? we don't need it any more now that we've set up an issues template